BST/BinaryTreeFromLowToHigh.py

```
@@ -0,0 +1,44 @@
+          self.right=None 
+class Solution:
+        def __init__(self):
+            self.array=[]
+            self.root=None
+        def preOrder(self,root:TreeNode):
+            if not root:
+                return root
+            self.array(root.val)
+            self.preOrder(root.left)
+            self.preOrder(root.right)
+        def insert(self,value):
```

Pylint (3.3.7)

[warning] Bad indentation. Found 8 spaces, expected 4

(W0311)